examples/resources/grafana_data_source/resource.tf

```diff
@@ -50,3 +50,16 @@ resource "grafana_data_source" "stackdriver" {
     private_key = "-----BEGIN PRIVATE KEY-----\nprivate-key\n-----END PRIVATE KEY-----\n"
   }
 }
+
+resource "grafana_data_source" "github" {
```

Comment:
Since this requires an external plugin, I wouldn't add it as an example here

Reply:
Understood thanks, I'll remove this.